js/game.js

```diff
@@ -0,0 +1,253 @@
+import {Player} from './players.js';
+import {Board} from './board.js';
+
+function returnAccumulatedScores(data) {
+
+  let dataToAccumulate = [];
+
+  do {
+
+    let elementToCompare = data.shift();
+    elementToCompare.apperances = 0;

```

Comment:
Nie wiem jak dokładnie wyglądają dane, które Ci tutaj spływają, ale czystszy zapis to:

`elementToCompare = { ...elementToCompare, apperances: 0 }`

Reply:
hmm, nie spotkalem sie z czyms takim, ale rozumiem ze rozrzucam key value pair po nowym obiekcie i dodaje kolejna?

Reply:
Tak, to jest podejście `immutability`, zamiast dodawać kolejne properties używając `dot notation` tworzysz nowy obiekt, który jest kopią oryginalnego obiektu i ma dodatkowe pole. To co napisałeś jest w pełni valid, po prostu ja osobiście lubię ograniczać mutowanie danych do minimum

Reply:
oki, czaje idee juz mi o tym mowiles, ale jak widac nie ejstem biegły :)